toolkit/scripts/toolchain/container/toolchain-remote-wget-list

```diff
@@ -17,7 +17,7 @@ http://ftp.gnu.org/gnu/make/make-4.2.1.tar.gz
 https://www.kernel.org/pub/linux/docs/man-pages/Archive/man-pages-5.02.tar.xz
 https://ftp.gnu.org/gnu/mpc/mpc-1.1.0.tar.gz
 http://www.mpfr.org/mpfr-4.0.1/mpfr-4.0.1.tar.xz
-https://invisible-mirror.net/archives/ncurses/ncurses-6.2.tar.gz
+https://invisible-mirror.net/archives/ncurses/current/ncurses-6.3-20220612.tgz
```

Comment:
Same comment as the URL in the cgmanifest

Reply:
same response as for the URL